qa/run_tests.sh

```diff
@@ -102,9 +102,10 @@ trap finally EXIT
 if [ ! -z $CPU_ONLY ] && [ $CPU_ONLY -eq 1 ]
 then
   pytest \
+    -v -rxXs \
```

Comment:
Do we always want `-v` in CI runs?

Reply:
My bad, I added it for local testing and accidentally committed it. Let me remove it.